libr/asm/arch/avr/format.c

```diff
@@ -26,7 +26,9 @@
 #include <stdio.h>
 #include <string.h>
 #include <stdlib.h>
+#include <stdbool.h>
```

Comment:
already included by r_util. as well as all the other stdio.string, stdlib includes, can be removed if you just include r_util.h"